lib/interface.ts

```diff
@@ -3,16 +3,22 @@
 import restify = require('restify');
 import bunyan = require('bunyan');
 import BAPI = require('./blpapi-wrapper');
+import Session = require('./apisession/session');
 
 export interface IOurRequest extends restify.Request {
     clientCert?: any;
     blpSession: BAPI.Session;
+    apiSession?: Session;
+}
+
+export interface IBufferedData<T> {
```

Comment:
Having `historical-buffer-manager` to only contains one class enable me to use the `export =` syntax, which is much user friendly to use(don't need to call the class name again and let the user define what it is called). And since we have this interface file to put all interfaces that will be used across our application, I figure this could be a good place.